### ISSUE TITLE
#1984127943------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#441]
<strong>Source:</strong>`<a aria-label="" aria-live="polite" class="button-padding--shrunk icon-end-empty" href="https://docdev.arcgis.com/en/iot/get-started/what-is-arcgis-velocity.htm" name="false" target="_blank" type="false">`
<strong>Element Location: </strong>["#help-button-get-started\\.global","a[name=\"false\"]"]
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attributeTags: [Ljava.lang.String;@4f2ac7e0
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
